app/controllers/conversations_controller.rb

```diff
@@ -0,0 +1,17 @@
+class ConversationsController < ApplicationController
+  def create
+
+  end
+
+  def new
+
+  end
+
+  def show
+    @message = Message.new
+    @conversation = Conversation.find_by(id: params[:id])
+    @match_user = MatchUser.where(creator_id: @conversation.sender_id, target_id: @conversation.recipient_id)
+    @all_messages = @conversation.messages
+    @recipient = User.find_by(id: @conversation.recipient_id)
+  end
```

Comment:
Also, once you have the associations, you don't need to set all these extra instance variables.